runtime/scheduler.hpp

```diff
@@ -133,7 +134,8 @@ class Scheduler
      *
      * \return              Pointer to the target task
      */
-    Scheduler_task* get_task_by_id(uint16_t task_id) const;
+    const Scheduler_task* get_task_by_id(uint16_t task_id) const;
+    Scheduler_task* get_task_by_id(uint16_t task_id);
```

Comment:
can't you add that explanation in the detail of the documentation section please ?